fix for:
error: class Foam::UList<unsigned int> Foam::UList<unsigned int>::UList is private within this context
see also https://sourceforge.net/p/foam-extend/tickets/68/
--- foam-extend-4.1-20210705/src/foam/containers/Lists/PackedList/PackedList.H.orig	2024-09-12 21:13:26.103914000 +0200
+++ foam-extend-4.1-20210705/src/foam/containers/Lists/PackedList/PackedList.H	2024-09-12 21:13:53.608924991 +0200
@@ -147,7 +147,7 @@
 class PackedList
 :
     public PackedListCore,
-    private List<unsigned int>
+    public List<unsigned int>
 {
 protected:
 
